.travis.yml

```diff
@@ -40,6 +40,8 @@ env:
 
 python:
   - "2.7"
+  - "3.3"
+  - "3.4"
```

Comment:
FWIW we decided some time ago to support Python 2.7 and 3.5 only, see https://github.com/inveniosoftware/invenio/issues/3647. Is Python 3.3 or 3.4 needed for B2SHARE? CC @nharraud